simpleSequential/cube.py

```diff
@@ -1 +1,3 @@
 # Create a function that take an integer number and return the cube of that number.
+def cube(n: int) -> int:
+    return n * n * n
```

Comment:
This is not scalable, what if I ask you to calculate n^20. There is an operation that you can use.